doc/sphinx/language/core/records.rst

```diff
@@ -53,8 +53,8 @@ expressions. In this sense, the :cmd:`Record` construction allows defining
    :cmd:`Record` and :cmd:`Structure` are synonyms.
 
    This command supports the :attr:`universes(polymorphic)`,
-   :attr:`universes(template)`, :attr:`universes(cumulative)`, and
-   :attr:`private(matching)` attributes.
+   :attr:`universes(template)`, :attr:`universes(cumulative)`,
+   :attr:`private(matching)`, and :attr:`projections(primitive)` attributes.
```

Comment:
Huh, I had no idea! :-)

Reply:
You got this backwards, @jfehrle: the Oxford comma is typical in AmE and somewhat less common in BrE; see https://en.wikipedia.org/wiki/Serial_comma

Reply:
OK now I am confused. I actually agree with @jfehrle on the comma convention here, since it is what I am used too. However, before I changed anything in this document, the attributes were listed with the extra `,`. Which one should be ought to use?

Reply:
@cpitclaudel Wikipedia is mistaken.  The Oxford comma is not commonly used in US English--not in journalism, not in general-interest books and not in the few academic papers I've read.  Nor have I have seen it much in the Coq documentation.

@Zimmi48 Any thoughts?

Reply:
> Wikipedia is mistaken.

I didn't check the references, but I'm not expecting this claim to be wrong:

>A majority of American style guides mandate use of the serial comma. These include: APA style,[8] The Chicago Manual of Style, Garner's Modern American Usage,[9] The MLA Style Manual, Strunk and White's The Elements of Style,[10] and the U.S. Government Printing Office Style Manual. By contrast, the Associated Press Stylebook and The New York Times Style Book[11] advise against it.

Also, for the record, by own grammar checking tool, which is set to American English also recommends it.

That being said, since this is apparently a debated topic, I would tend to side with Jim against the Oxford comma. But I would also not lose much time over this.

Reply:
@jfehrle Well, I'm no expert, so I usually go by published sources; here's what my copy of "Garner's Modern English Usage" has to say (sorry, I don't have an electronic copy):

![image](https://user-images.githubusercontent.com/8181630/127564792-f5ad7da3-be63-4beb-8ea1-29660d91e301.png)

![image](https://user-images.githubusercontent.com/8181630/127564963-c013700f-779d-413b-8538-cc7d0a3d85f9.png)

FWIW, Coq is copyrighted to `The Coq development team, INRIA, CNRS, and contributors`

Reply:
@cpitclaudel I'm familiar with the discussion.  It's just that the Oxford comma is rarely used in practice.  As Wikipedia notes, the US style guides are not unanimous.  Actual usage trumps any "authoritative" source.

The part about "When the members are compound, calling for _and_ within themselves, clarity demands the final comma." would be the main exception in my experience.

Reply:
Eh, we have actual usage right in front of us: the way the manual is currently written.  So I see no need to change it to conform to perceptions about what's more or less commonly used out there.

Reply:
I had a look back through the git blame and it appears that @Zimmi48 wrote it first in the style @jfehrle suggests, it seems later @ejgallego changed it to have the `,` before the `and`.

Reply:
@cpitclaudel I would not select the documentation as an exemplar of excellent, well-edited writing from which you can infer what is common usage.

Reply:
Indeed not, and hence there's no need to get into these long arguments about comma usage preferences :) Sorry that I got dragged into this — I was originally reacting only to the comment that this would "generally be considered incorrect", which isn't true, AFAICT.

Reply:
Indeed, "incorrect" was a bit too emphatic.

Reply:
I suggest settling for inconsistency on this topic, that is, let the PR author do what they want, and avoid large-scale changes relative to these commas.

Reply:
> I had a look back through the git blame and it appears that @Zimmi48 wrote it first in the style @jfehrle suggests, it seems later @ejgallego changed it to have the `,` before the `and`.

That is: there is no Oxford comma in all copyright statements (manual and header files) but they appear in the author fields of the `.opam` files. Not really a "change" IMHO.